rfcs/proposed/task_group_dynamic_dependencies/README.md

```diff
@@ -0,0 +1,311 @@
+# Extending task_group to manage dynamic dependencies between tasks
+
+## Introduction
+
+Back in 2021, during the move from TBB 2020 to the first release of oneTBB, 
+the lowest level tasking interface changed significantly and was no longer 
+promoted as a user-facing feature. Instead, the guidance since then has been 
+to use the task_group or the flow graph APIs to express patterns that were 
+previously expressed using with the lowest level tasking API. And for most 
+cases, this has been sufficient. However, there are some use cases that create 
+dynamically expanding graphs of dependent tasks are awkward to express using 
```

Comment:
I don't get this grammar. And I think awkward is likely not a good wording.

Reply:
Suggestion: However, there is one use case which is not straightforward to express by the revised API: Dynamic task graphs which are not trees.

Reply:
Fixed